src/data/taxonomy-redirects.json

```diff
@@ -600,7 +608,7 @@
     ]
   },
   {
-    "url": "/docs/apm/agents",
+    "url": "https://developer.newrelic.com/instant-observability/?search=&category=application-monitoring",
```

Comment:
@paperclypse , Why are we sending users to the developer site?

Reply:
Because there isn't a good generic "agents" doc page.